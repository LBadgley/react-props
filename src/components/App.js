import React from 'react';
import Dog from './Dog';
import Header from './Header';


export default function App() {
  console.log('that is my dog, maaaan');
  return (
    <>
      <Header />
      <Dog name="Ruby" age="2" weight="7.5 lbs"/>
    </>
  );
}
