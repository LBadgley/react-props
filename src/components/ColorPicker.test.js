import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('Color component', () => {
  it('renders a color', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
