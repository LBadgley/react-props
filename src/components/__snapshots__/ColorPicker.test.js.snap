// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color component renders a color 1`] = `<ColorPicker />`;
